examples/storm-jms-examples/src/main/java/org/apache/storm/jms/example/GenericBolt.java

```diff
@@ -15,27 +15,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.storm.jms.example;
 
 import java.util.Map;
 
-import org.apache.storm.topology.base.BaseRichBolt;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import org.apache.storm.task.OutputCollector;
 import org.apache.storm.task.TopologyContext;
 import org.apache.storm.topology.OutputFieldsDeclarer;
+import org.apache.storm.topology.base.BaseRichBolt;
 import org.apache.storm.tuple.Fields;
 import org.apache.storm.tuple.Tuple;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * A generic <code>org.apache.storm.topology.IRichBolt</code> implementation
  * for testing/debugging the Storm JMS Spout and example topologies.
- * <p>
- * For debugging purposes, set the log level of the
+ *
+ * <p>For debugging purposes, set the log level of the
  * <code>org.apache.storm.contrib.jms</code> package to DEBUG for debugging
- * output.
+ * output.</p>
```

Comment:
nit: I don't think p tags need to be closed.

Reply:
They're not necessary in HTML, but possible. I don't like the idea and it's been changed in XHTML. Do you insist?

Reply:
No, it's fine.